main.go

```diff
@@ -34,9 +36,51 @@ func main() {
 	if version == "" {
 		version = "2-dev"
 	}
+	// Orange to Yellow gradient colors (reversed order)
+	gradientColors := []string{
+
+		"\033[38;5;208m", // Orange
+		"\033[38;5;208m", // Orange (repeat for consistency)
+		"\033[38;5;226m", // Lighter Yellow-Orange
+
+	}
 	utils.Version = version
 	if binaryToDocker := os.Getenv("BINARY_TO_DOCKER"); binaryToDocker != "true" {
-		fmt.Println(logo, " ")
+		// ANSI escape code to reset color
+		const reset = "\033[0m"
+
+		// Print each line of the logo with a different color from the gradient
+		lines := strings.Split(logo, "\n")
+		fmt.Println(len(lines))
+		for i, line := range lines {
```

Comment:
seems to be unnecessary

Reply:
Regarding line 54 which is printing right?
I removed it.